src/android.ts

```diff
@@ -156,14 +160,7 @@ export class AndroidRobot implements Robot {
 	}
 
 	public async getElementsOnScreen(): Promise<ScreenElement[]> {
-		const dump = this.adb("exec-out", "uiautomator", "dump", "/dev/tty");
-
-		const parser = new xml.XMLParser({
-			ignoreAttributes: false,
-			attributeNamePrefix: ""
-		});
-
-		const parsedXml = parser.parse(dump) as UiAutomatorXml;
```

Comment:
`memo`
moved out this chunk of code into `getParsedXml`. The reason being, when I perform a D-Pad action on AndroidTV, there might be changes in UI and I will need the latest information to know my current focused item.